platform/backend/backend.md

```diff
@@ -54,27 +78,154 @@ We don’t use Windows as OS for development machines. We use latest macOS or Ub
+
+### Authorization
+
+* [pundit](https://github.com/elabs/pundit) - Minimal authorization through OO design and pure Ruby classes.
+* [rolify](https://github.com/RolifyCommunity/rolify) - Role management library with resource scoping.
+
+### Active Record
+
+* [AASM](https://github.com/aasm/aasm) - State machines for Ruby classes.
+* [active_record_union](https://github.com/brianhempel/active_record_union) - UNIONs in ActiveRecord! Adds proper union and union_all methods to ActiveRecord::Relation.
+* [PaperTrail](https://github.com/paper-trail-gem/paper_trail) - PaperTrail lets you track changes to your models' data. It's good for auditing or versioning.
+* [paranoia](https://github.com/rubysherpas/paranoia) - ActiveRecord plugin allowing you to hide and restore records without actually deleting them.
+* [globalize](https://github.com/globalize/globalize) - Rails I18n de-facto standard library for ActiveRecord model/data translation.
+
+### Abstraction
+
+* [interactor](https://github.com/collectiveidea/interactor) - Interactor provides a common interface for performing complex user interactions.
+* [wisper](https://github.com/krisleech/wisper) - A micro library providing objects with Publish-Subscribe capabilities.
+
+### Pagination
+
+* [Pagy](https://github.com/ddnexus/pagy) - Pagy is the ultimate pagination gem that outperforms the others in each and every benchmark and comparison.
+* [Kaminari](https://github.com/amatsuda/kaminari) - A Scope & Engine based, clean, powerful, customizable and sophisticated paginator for modern web app frameworks and ORMs.
+
+### API
+
+* [rspec-rails-swagger](https://github.com/drewish/rspec-rails-swagger) - This gem helps you generate Swagger docs by using RSpec to document the paths. 
+* [swagger-blocks](https://github.com/fotinakis/swagger-blocks) - For generating API documentation in [Swagger](http://swagger.io/specification/) format.
+
+## Email
+
+* [letter_opener](https://github.com/ryanb/letter_opener) - Preview mail in the browser instead of sending.
+
+## Decorators
+
+* [Draper](https://github.com/drapergem/draper) - Draper adds an object-oriented layer of presentation logic to your Rails application.
+
+## File Uploading
+
+* [shrine](https://github.com/janko-m/shrine) - File Attachment toolkit for Ruby applications.
+    * [image_processing](https://github.com/janko/image_processing) - High-level image processing helper methods with libvips and ImageMagick/GraphicsMagick.
+    * [shrine-url](https://github.com/shrinerb/shrine-url) - Custom URL storage for Shrine.
+
+## HTTP Clients
+
+* [Faraday](https://github.com/lostisland/faraday) - HTTP client lib that provides a common interface over many adapters (such as Net::HTTP) and embraces the concept of Rack middleware when processing the request/response cycle.
+
+## Searching
+
+* [searchkick](https://github.com/ankane/searchkick) - Intelligent search made easy with Rails and Elasticsearch.
```

Comment:
What about https://github.com/elastic/elasticsearch-rails? It is listed on Elasticsearch Rails integration page: https://www.elastic.co/guide/en/elasticsearch/client/ruby-api/current/ruby_on_rails.html.

Reply:
@TeTiRoss, Ok. It's more common solution.